modules/local/locidex_select.nf

```diff
@@ -39,74 +39,109 @@ process LOCIDEX_SELECT {
 
     input:
     tuple val(meta), val(top_hit), val(contigs)
+    // TODO update locidex to allow for file passed in
```

Comment:
another sneaky TODO 😸

Reply:
one of these days, I will remember to actualy remove the todos when I do them 😢 

fixed in: https://github.com/phac-nml/mikrokondo/pull/96/commits/e938535aa3d745d843dfc1eb621100c6210e5452